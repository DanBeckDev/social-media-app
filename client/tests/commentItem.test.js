import React from 'react'
import { shallow } from 'enzyme'
import CommentItem from '../components/commentItem'

it('expect to render Comment Component', () => {
  expect(shallow(<CommentItem />)).toMatchSnapshot()
})