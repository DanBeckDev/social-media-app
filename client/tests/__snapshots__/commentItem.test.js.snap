// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Comment Component 1`] = `
<Comment>
  <CommentAvatar
    src="http://localhost:443/images/avatars/undefined"
  />
  <CommentContent>
    <CommentAuthor
      as="a"
    />
    <CommentMetadata>
      <div>
        NaN days ago
      </div>
    </CommentMetadata>
    <CommentText />
    <CommentActions>
      <CommentAction
        as="a"
      >
        Reply
      </CommentAction>
    </CommentActions>
  </CommentContent>
</Comment>
`;
